README.md

```diff
@@ -344,6 +344,18 @@ const { data: app } = await requestWithAuth(
       Used for internal logging. Defaults to <a href="https://developer.mozilla.org/en-US/docs/Web/API/console"><code>console</code></a>.
     </td>
   </tr>
+  </tr>
+    <th align=left>
+      <code>options.request.parseSuccessResponseBody</code>
+    </th>
+    <td>
+      <code>boolean</code>
+    </td>
+    <td>
+      If set to <code>false</code> the request will return a Stream as response.data for successful requests (e.g. <code>200 OK</code> status code).
```

Comment:
I actually wanted to add a "Useful in case you want to stream a file download." But I didnt know if you want such a opinionated comment in the documentation. But I fear that other devs will not immediatly understand why that is useful.

Reply:
How about the following wording:
This option may be useful when downloading field from the GitHub API

Reply:
do you mean "when downloading files"?

Reply:
Yeah. Bad autocorrect

Reply:
```suggestion
      If set to <code>false</code> the request will return a Stream as response.data for successful requests (e.g. <code>200 OK</code> status code).
      This option may be useful when downloading files from the GitHub API.
```